content/en/docs/develop/porting.md

```diff
@@ -109,25 +108,25 @@ The different ways to do this are covered later in this tutorial.
 ### Creating a Boilerplate Microlibrary for Your Application
 
 To get started, we must create a new library for our application.
-The premise here is that we are going to "wrap" or "decorate" the source code of `iperf3` with the _lingua franca_ of Unikraft's build system.
-That is, when we eventually build the application, the Unikraft build system will understand where to get the source code files, which ones to compile and how, with respect to the rest of Unikraft's internals and other dependencies.
+The premise here is that we are going to "wrap" or "decorate" the source code of `iperf3` with the _lingua franca_ of Unikraft's build tool.
+That is, when we eventually build the application, the Unikraft build tool will understand where to get the source code files, which ones to compile and how, with respect to the rest of Unikraft's internals and other dependencies.
 
 Let's first start by initializing a working environment for ourselves:
 
 1. Let's create a workspace with a typical Unikraft structure using `kraft`:
 
    ```bash
-   $ cd ~/workspace
-   $ export UK_WORKDIR=$(pwd)
-   $ kraft list update
-   $ kraft list pull unikraft@staging
+   cd ~/workspace
+   export UK_WORKDIR=$(pwd)
+   kraft list update
+   kraft list pull unikraft@staging
    ```
 
    This will generate the necessary directory structure to build a new Unikraft application, and will also download the latest `staging` branch of Unikraft's core.
    When we list the directories, we should get something like this:
 
    ```bash
-   tree -L 1
+   $ tree -L 1
    .
    ├── apps
```

Comment:
Provide the output using `tree -L 1 --charset=ascii` to make it ASCII. Leave the `--charset=ascii` from the command (people don't need to type that), but make the output ASCII. I did that [here](https://github.com/unikraft-upb/scripts/blob/main/README.md).